bin/setup_venv.sh

```diff
@@ -195,12 +194,15 @@ else
 fi
 
 # Install general package requirements
-pip3 install --requirement "${script_dir}/../pulumi/python/requirements.txt"
+PIPENV_VERBOSITY=-1 PIPENV_PIPFILE="${script_dir}/../pulumi/python/Pipfile" pipenv install
```

Comment:
**annotation:**
Here we are:

1. Silencing an informational message telling us that pipenv is going to use the current venv with `PIPENV_VERBOSITY=-1`
2. Pointing `pipenv` at our `Pipfile` which is not in the current dir with `PIPENV_PIPFILE`
3. Installing all packages